src/worker/core/player/genRatings.football.ts

```diff
@@ -357,6 +357,43 @@ const genRatings = (
 		pots: { ...defaultOvrsOrPots },
 	};
 
+	// Youngest prospects !== 21 will be scaled, scaling stops at age 14 and 30
+	let age = g.get("draftAges")[0];
+	age = helpers.bound(age, 14, 30);
```

Comment:
I understand why you want an upper bound (because at some age, players stop development, so continuing to boost them after that would make super players).

But why do you want a lower bound? No matter how low a player's age, he's still going to be improving every year.

Reply:
it is subjective, but ages under 10 have a lot of 0 ovr players, so it's not rly fun if you want to do some little kid league and have to deal with most of the players being incompetent (maybe realistic). Bigger thing, if everyone becomes 0 OVR, everyone will end up developing in very similar styles and only be separated by height rating.

Reply:
you could lower it or remove it if you believe its better, i just didn't really like it without a min limit